src/gt4py/backend/templates/stencil_module.py.in

```diff
@@ -117,6 +117,10 @@ class {{ class_name }}(StencilObject):
     ):
         if exec_info is not None:
             exec_info["call_start_time"] = time.perf_counter()
+            if exec_info.setdefault("__aggregate_data", False):
+                stencil_info = exec_info.setdefault("{{ class_name }}", {})
+                stencil_info["call_start_time"] = exec_info["call_start_time"]
+
```

Comment:
I think this part can be merged with the next conditional, which makes the code simpler to understand and slightly faster since we skip one conditional:

```suggestion
```

Reply:
Yes, definitely. I'll do it.